### PR TITLE
Update nginx images in thanos

### PR DESCRIPTION
**Pull request by loganhz**
https://github.com/cnrancher/pandaria/issues/724

### CODE REVIEW

File: charts/rancher-thanos/v0.0.2/values.yaml [hunk 1]
```diff
@@ -21,8 +21,8 @@ grafana:
     tag: 6.3.6
     pullPolicy: IfNotPresent
     proxy:
-      repository: rancher/nginx
-      tag: 1.17.4-alpine
+      repository: cnrancher/nginx
+      tag: stable
```
Comment (gitlawr) [inline charts/rancher-thanos/v0.0.2/values.yaml]: Use a specific version considering that we might need to update it in follow-up releases.

File: charts/rancher-thanos/v0.0.2/values.yaml [hunk 2]
```diff
@@ -122,7 +122,7 @@ ui:
   replicaCount: 1
   image:
     repository: cnrancher/global-monitoring-ui
-    tag: 0.0.2
+    tag: 0.0.3
```
Comment (gitlawr) [inline charts/rancher-thanos/v0.0.2/values.yaml:125]: What is it for?

File: charts/rancher-thanos/v0.0.2/values.yaml (hunk 1 above)
Comment (loganhz) [reply charts/rancher-thanos/v0.0.2/values.yaml]: fixed

File: charts/rancher-thanos/v0.0.2/values.yaml (hunk 2 above)
Comment (loganhz) [reply charts/rancher-thanos/v0.0.2/values.yaml:125]: it was also using alpine. Updated base image here

**Review by gitlawr — APPROVED** [review]
LGTM

**Pull request merged**